anime/views.py

```diff
@@ -87,3 +89,30 @@ def anime_delete(id: int):
     db.session.commit()
 
     return '', 204
+
+@animes_bp.route('/api/v1/animes/', methods=['POST'])
+def create_anime():
+    """Route for inserting a new anime resource to DB
+
+    :return: returns successful anime input
+    :rtype: AnimeSchema
+    """
+
+    anime_schema = AnimeSchema()
+    try:
+        schema_dict = anime_schema.loads(request.data)
+    except ValidationError as e:
+        return {"error": e.messages}, 400
+
+    try:
+        db.session.add(Anime(**schema_dict))
+        db.session.commit()
+    except IntegrityError:
+        db.session.rollback()
+        return {
+            "error": "sqlite3.IntegrityError: UNIQUE constraint failed: animes.anime_id",
+            "message": "Anime ID exists already in the database",
+            "detail": "Ensure that anime ID is unique"
+            }, 400
```

Comment:
The error handler should simplify this part because, you will be able to use the message raised from the exception and return it in the response.